internal/runners/secrets/set.go

```diff
@@ -75,6 +75,8 @@ func (s *Set) Run(params SetRunParams) error {
 		return err
 	}
 
+	s.out.Print(output.Prepare("", &struct{}{})) // {} for JSON
```

Comment:
Another example of a command that just does not produce any JSON output, so we should just mark it as does not support.